### PR TITLE
dev-libs/libezV24: fix LICENSE, EAPI=7 bump

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This fixes the LICENSE and adds a EAPI=7 ebuild. Please review.

### CODE REVIEW

File: dev-libs/libezV24/libezV24-0.1.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+src_prepare() {
+	tc-export AR CC RANLIB
+	default
+	sed -i -e 's:__LINUX__:__linux__:' *.c *.h || die
+}
+
+src_install() {
+	export NO_LDCONFIG="stupid"
+	emake DESTDIR="${D}" LIBDIR="/usr/$(get_libdir)" install
```
Comment (mgorny) [inline dev-libs/libezV24/libezV24-0.1.1-r1.ebuild]: Some EPREFIX support maybe, while you're bumping the EAPI?

Comment (mm1ke) [reply dev-libs/libezV24/libezV24-0.1.1-r1.ebuild]: done :)

Comment (mgorny) [reply dev-libs/libezV24/libezV24-0.1.1-r1.ebuild]: I'm going to guess there's a `PREFIX` variable that you need to adjust as well.

Comment (mm1ke) [reply dev-libs/libezV24/libezV24-0.1.1-r1.ebuild]: @mgorny 
I'm sorry, can you explain. I don't know exactly what you mean.

Comment (mgorny) [reply dev-libs/libezV24/libezV24-0.1.1-r1.ebuild]: I mean that there's probably a variable defaulting to `/usr` where you need to append `${EPREFIX}` so it'll apply to everything. Hm, unless the library is the only thing it ever installs.

Comment (mm1ke) [reply dev-libs/libezV24/libezV24-0.1.1-r1.ebuild]: @mgorny 
Thx for explaining and sorry for the late reply. I've looked at the Makefile and as you said there is a ```PREFIX``` variable, which i'm now also adjusting with ```EPREFIX``` too.

**Pull request closed without merging**